WooCommerce/Classes/ViewRelated/Orders/Order Creation/PaymentSection/ShippingLineDetails.swift

```diff
@@ -102,6 +96,38 @@ struct ShippingLineDetails: View {
     }
 }
 
+/// Adds a currency symbol to the left or right of the provided content
+///
+private struct CurrencySymbol: ViewModifier {
+    let symbol: String
+    let position: CurrencySettings.CurrencyPosition
+
+    func body(content: Content) -> some View {
+        HStack {
+            switch position {
+            case .left, .leftSpace:
+                Text(symbol)
+                    .bodyStyle()
+                content
+            case .right, .rightSpace:
+                content
+                Text(symbol)
+                    .bodyStyle()
+            }
```

Comment:
Can we handle different spacing cases? With `spacing: .zero` in `HStack` and additional spacer views.
<img width=350 src=https://user-images.githubusercontent.com/3132438/155110294-49c69418-bc72-4cdd-aa32-897b4bb79971.png>